c7n/tags.py

```diff
@@ -263,6 +274,11 @@ def __call__(self, i):
         tag = self.data.get('tag', DEFAULT_TAG)
         op = self.data.get('op', 'stop')
         skew = self.data.get('skew', 0)
+        skew_hours = self.data.get('skew_hours', 0)
+        tz = zoneinfo.gettz(Time.TZ_ALIASES.get(self.data.get('tz', 'gmt')))
+        if not tz:
```

Comment:
this should ideally be a validation error pre policy execution instead of a runtime error, ie. do this check in the validate method.

Reply:
oh! yes, that makes sense.